content-team/working-docs/social-media-policy.md

```diff
@@ -0,0 +1,90 @@
+# Social Media Strategy and Best Practices
+DD&I Content Team
+
+## Channels
```

Comment:
It might be a good idea to have each of these as links to the respective channel.